Core/include/Acts/EventData/TrackProxy.hpp

```diff
@@ -687,14 +687,20 @@ class TrackProxy {
 
       // reverse using standard linked list reversal algorithm
       reverseTrackStates();
+    } else {
+      // just copy the track state indices
+      tipIndex() = other.tipIndex();
+      stemIndex() = other.stemIndex();
```

Comment:
These are only valid if the track is copied within the same container though, right? If you do this generically, you might get corrupted tracks.

I see how this can be useful as a "shallow copy", however.